src/main/java/rs117/hd/HdPluginConfig.java

```diff
@@ -127,19 +127,19 @@ default UIScalingMode uiScalingMode()
 	@ConfigItem(
 		keyName = "anisotropicFilteringLevel",
 		name = "Anisotropic Filtering",
-		description = "Configures the anisotropic filtering level.",
+		description = "Configures the anisotropic filtering level from 0 to 16x.",
 		position = 4,
 		section = generalSettings
 	)
 	default int anisotropicFilteringLevel()
 	{
-		return 1;
+		return 16;
 	}
 
 	@ConfigItem(
 		keyName = "unlockFps",
 		name = "Unlock FPS",
-		description = "Removes the 50 FPS cap for camera movement",
+		description = "Removes the 50 FPS cap for camera movement, lighting and shadows.",
```

Comment:
I'd maybe change this to something along the lines of "Removes the 50 FPS cap. Animations are still capped." I think it's really anything that's tied to client ticks that's still capped, but I'm not sure about what is and what isn't.

Reply:
How about "Removes the 50 FPS cap for some game content such as camera movement and dynamic lighting."?

Some people have been setting/advising caps under the bad assumption that only the camera is changing so i wanted to hint at more than that going on.

Reply:
That sounds better, yeah. It doesn't include things like the animated water effects, but I suppose the list shouldn't have to be exhaustive.